acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -64,7 +72,12 @@
   }
 
   teardown do
-    apply_manifest_on(agent, manifest_uninstall_httpd)
+    if platform == 'sles'
+      on agent, 'rpm -e apache2 apache2-worker',  :accept_all_exit_codes => true
+      on agent, 'rpm -e apache2 apache2-prefork', :accept_all_exit_codes => true
```

Comment:
It looks like apache2 is also bringing in `libapr1` and `libapr-util1`, which maybe we don't care? Also it looks like this might work:

```
xyrtkhd4e08zstn:~ # zypper remove -y apache2 apache2-prefork apache2-worker libapr1 libapr-util1
Restoring system sources...
Parsing metadata for SLES10-SP4...
Parsing metadata for SLES10-SP4-Updates...
Parsing RPM database...
package 'apache2-prefork' not found
Summary:
<uninstall> [S0:0][package]libapr-util1-1.2.2-13.13.18.x86_64
<uninstall> [S0:0][package]apache2-worker-2.2.3-16.48.1.x86_64
<uninstall> [S0:0][package]libapr1-1.2.2-13.10.1.x86_64
<uninstall> [S0:0][package]apache2-2.2.3-16.48.1.x86_64
Removing: [S0:0][package]apache2-worker-2.2.3-16.48.1.x86_64
Removing: [S0:0][package]apache2-2.2.3-16.48.1.x86_64
Removing: [S0:0][package]libapr-util1-1.2.2-13.13.18.x86_64
Removing: [S0:0][package]libapr1-1.2.2-13.10.1.x86_64
xyrtkhd4e08zstn:~ # echo $?
0
```